tests/models/test_dagbag.py

```diff
@@ -837,6 +837,7 @@ def _sync_perms():
             mock_sync_perm_for_dag.assert_called_once_with(
                 "test_example_bash_operator", {"Public": {"can_read"}}
             )
+        delete_dag_specific_permissions()
```

Comment:
Can you look at creating a pytest fixture for this instead? Then it'll at least run if something goes wrong in this test.

Reply:
This suite uses unittest and hence added to setup and clear down instead. Also. moved the function `as is` into `db.py` module under `test_utils` for consistency and removed `permissions.py` as it's not required anymore. Is that OK?